addons/registry-route/registry-route.addon

```diff
@@ -16,4 +16,9 @@ docker cp origin:/var/lib/origin/openshift.local.config/master/ca.crt /etc/docke
 echo Add-on '#{addon-name}' created docker-registry route. Please run following commands to login to the OpenShift docker registry:
 echo $ eval $(minishift docker-env)
 echo $ eval $(minishift oc-env)
+echo
+echo If your deployed version of OpenShift is < 3.7.0 use.
 echo $ docker login -u developer -p `oc whoami -t` docker-registry-default.#{routing-suffix}:443
+echo
+echo If your deployed version of OpenShift is >= 3.7.0 use.
+echo $ docker login -u developer -p `oc whoami -t` docker-registry-default.#{routing-suffix}
```

Comment:
+1 works for me.

Reply:
This information will be only shown when the add-on is applied, but for this particular addon we need to enable it and then start minishift.

Reply:
@anjannath We will update the documentation about it in #2060

Reply:
@anjannath This information should be displayed if a user try to apply or enable it. For this add-on also you can use it as apply.